docs/guides/nrfconnect_factory_data_configuration.md

```diff
@@ -151,6 +152,24 @@ In the factory data set, the following formats are used:
 -   All certificates stored in factory data are provided in the
     [X.509](https://www.itu.int/rec/T-REC-X.509-201910-I/en) format.
 
+#### The appearance field description
+
+The `appearance` field in the factory data set describes device's visible appearance and it consist of the following attributes:
+
+- `finish` - This field is a uint8 number and indicates the visible finish of the product. 
+It refers to the `ProductFinishEnum` enum, and currently, it can be set in the following way:
+    - 0 - the product has a matte finish.
+    - 1 - the product has a satin finish.
+    - 2 - the product has a shiny or polished finish.
+- `primary_color_hue` - This field is the 16-bit integer and it represents the HSV color space hue of the device's color, which is the most representative. 
```

Comment:
How did you get information about this saturation, hue and etc? I saw the PR to spec, but there is only ColorEnum that describes primary color in RGB. I didn't found any information about the sub-fields.